pkg/dss/auth/auth_test.go

```diff
@@ -130,40 +130,40 @@ func TestRSAAuthInterceptor(t *testing.T) {
 }
 
 func TestMissingScopes(t *testing.T) {
-	ac := &Authorizer{requiredScopes: map[string][]string{
-		"PutFoo": {"required1", "required2"},
+	ac := &Authorizer{requiredScopes: map[Operation][]Scope{
+		"/dss/syncservice/PutFoo": {Scope("required1"), Scope("required2")},
 	}}
 
 	var tests = []struct {
 		info   *grpc.UnaryServerInfo
-		scopes map[string]struct{}
+		scopes map[Scope]struct{}
 		want   error
 	}{
 		{
 			&grpc.UnaryServerInfo{FullMethod: "/dss/syncservice/PutFoo"},
```

Comment:
Let's take a note to make these FullMethod names more similar to real FullMethod values (like, `/dss.SyncService/PutFoo` I think)